loadtest/mock-oauth-cfn.yml

```diff
@@ -119,28 +119,23 @@ Resources:
           }
 
           const HANDLERS = {
-
-            'GET:/config': function(event, context, callback) {
-              return callback(null, {
-                statusCode: 200,
-                headers: {
-                  "content-type": "application/json"
-                },
-                body: '{ "browserid": { "issuer": "${MockIssuer}" } }'
-              });
-            },
-
             'POST:/v1/verify': function(event, context, callback) {
               try {
                 const token = JSON.parse(event.body).token;
                 const mockResponse = JSON.parse(token);
+                const mockStatus = mockResponse.status || 200;
+                const mockBody = mockResponse.body || {};
+                // Ensure successful responses always come from the mock issuer.
+                if (mockStatus < 400) {
+                  mockBody.issuer = "${MockIssuer}";
```

Comment:
Does anything rely on the old nested format?

Reply:
Sorry, not sure what you mean by "the old nested format" here.

Reply:
Oh, sorry, I meant how the old response was `{ "browserid": { "issuer": "${MockIssuer}" } }'`, but the new response sets `mockBody.issuer`. Does it need to be `mockBody.browserid.issuer`?

Reply:
>  old response was `{ "browserid": { "issuer": "${MockIssuer}" } }`

That was for the `/config` endpoint, which we're no longer modifying. The new code is adding an explicit `issuer` field to the `/v1/verify` response (which previously would have been provided implicitly by the `mockBody`, if it had such a field).

Reply:
Derp. I...don’t know how I missed that it was for a completely different endpoint. :scream: Thanks!